src/renderer/components/+workloads-overview/overview.tsx

```diff
@@ -61,9 +59,10 @@ export class WorkloadsOverview extends React.Component<Props> {
     if (isAllowedResource("events")) {
       stores.push(eventStore);
     }
```

Comment:
```suggestion
    if (isAllowedResource("events") && !this.isUnmounting) {
      await eventStore.loadAll()
      this.disposers.push(eventStore.subscribe());
    }
```

Though TBH I think that `subscribe` should `loadAll` if it hasn't (which in and of itself is also an optimization).

Reply:
Maybe but that's out of scope for this PR 🙂

Reply:
True, but I definitely believe that awaiting the load and pushing the subscribe in each of the `if`'s is within scope.

Reply:
Ok, I somehow missed the point in "suggested change". I agree, would be better to do both loadAll & subscribe within if.